app/helpers/application_helper.rb

```diff
@@ -201,4 +201,8 @@ def filters_params
     end
     filters_params
   end
+
+  def format_uuid(uuid)
+    uuid[0,4]+"…"+uuid[uuid.length-4,uuid.length]
```

Comment:
This is a bit more concise and idiomatic:

```suggestion
    "#{uuid[0,4]}…#{uuid[-4,4]}"
```